app/models/service.rb

```diff
@@ -22,6 +22,7 @@ class Service < ActiveRecord::Base
   accepts_nested_attributes_for :addresses
```

Comment:
I think we will want to add `accepts_nested_attributes_for :instructions` here. At least when creating new services (updating them is another story), it seems that our pattern is to include the attributes of any child models in the payload (as we do with addresses, schedules, and notes).

Reply:
No harm in adding that, I guess. But aren't we going to persist these separately through dedicated endpoints?

Reply:
Yes – I can look into using the dedicated endpoints for both creating new instructions on services. But at the least adding the accepts nested attributes will hold us over in the meantime. 

I will definitely be using the dedicated endpoints for updating instructions on services (which sort of follows our current pattern except I don't think we will be using change requests for instructions updates).